android/expoview/build.gradle

```diff
@@ -208,6 +208,7 @@ dependencies {
   api project(':expo-background-fetch')
   api project(':expo-barcode-scanner')
   api project(':expo-barcode-scanner-interface')
+  api project(':expo-bluetooth')
```

Comment:
@ide should we remove expo-bluetooth from expo client until it's more stable? Seeing as it adds another permission we may not be able to add it anyways.

Reply:
Yes, let's leave it out and not touch permissions.